src/Nethermind/Nethermind.Serialization.Json/StreamPipeWriter.cs

```diff
@@ -42,6 +43,12 @@ public void Advance(int count)
     public Memory<byte> GetMemory(int sizeHint = 0) => _writer.GetMemory(sizeHint);
 
     public Span<byte> GetSpan(int sizeHint = 0) => _writer.GetSpan(sizeHint);
+
+    public ValueTask CompleteAsync(Exception? exception = null)
```

Comment:
what is the purpose of Exception here?

Reply:
it is an overrided method

Reply:
![image](https://github.com/NethermindEth/nethermind/assets/33181301/2e88402e-342f-48b6-b6e2-65dc19e28d1a)

Reply:
I do not see any overrides here, you are implementing `ICountingBufferWriter`

Reply:
two classes are implementing `ICountingBufferWriter` and one of them has this method as an override to the one in `PipeWriter`